wolfcrypt/src/tfm.c

```diff
@@ -5305,7 +5305,8 @@ static int fp_read_radix_16(fp_int *a, const char *str)
       j &= DIGIT_BIT - 1;
   }
 
-  a->used = k + 1;
+  a->used = k + (j != 0 ? 1 : 0);
```

Comment:
Better if k updated before check for too large rather than at end of loop.

Reply:
Force pushed an update to resolve this.